src/Analyzer.Core.UnitTests/templates/ParameterPassingFail.json

```diff
@@ -46,9 +46,6 @@
               "type": "string"
             }
           },
-          "variables": {
```

Comment:
Was this removed by accident?

Reply:
It's the analog to the comment you made on the bicep version of this file (in the module specifically). The bicep file was decompiled from this template and inherited the unnecessary variable.

Reply:
Oh, makes sense